src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,527 @@
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Number of Workers
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Enter a number or a range for the number of
+                                    people employed at the location. For
+                                    example: 100, 100-150.
+                                </Typography>
+                                <StyledSelect
+                                    id="number_of_workers"
+                                    label={null}
+                                    name="Number of Workers"
+                                    options={numberOfWorkersOptions || []}
+                                    value={numberOfWorkers}
+                                    onChange={handleNumberOfWorkers}
+                                    placeholder="Enter the number of workers as a number or range"
+                                    aria-label="Number of workers"
+                                    isMulti={false}
+                                    className={`basic-multi-select notranslate ${classes.selectStyles}`}
+                                    styles={selectStyles}
+                                />
+                            </div>
+                            <div className={classes.inputSectionWrapStyles}>
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Parent Company
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Select or enter the company that holds
+                                    majority ownership for this production.
+                                </Typography>
+                                <StyledSelect
+                                    creatable
+                                    label={null}
+                                    name="Parent company"
+                                    value={parentCompany}
+                                    onChange={handleParentCompany}
+                                    placeholder="Select"
+                                    aria-label="Parent company"
+                                    className={`basic-multi-select notranslate ${classes.selectStyles}`}
```

Comment:
className already exists in the StyledSelect.

Reply:
edit classes